pkg/analyzer/lfp_npm.go

```diff
@@ -284,6 +284,17 @@ func npmIsUrlFollowsPathConvention(sourceUrl string, pkg string, trustedUrls []s
 
 	// Example: @angular/core from https://registry.npmjs.org/@angular/core/-/core-1.0.0.tgz
 	scopedPackageName := strings.Split(path, "/-/")[0]
+	if slices.Contains(acceptablePackageNames, scopedPackageName) {
+		return true
+	} else {
```

Comment:
Can we simplify this? Instead of using `else`, just fall through to the `for` loop to check for acceptable conditions. If nothing matches, `return false` as the default fallback.